app/views/users/rdvs/index.html.slim

```diff
@@ -26,3 +26,7 @@
             i.far.fa-calendar.fa-stack-1x.text-white
           .text-center.mt-2
             = add_button 'Prendre un RDV', root_path
+
+.text-center.my-5
+  a href="https://voxusagers.numerique.gouv.fr/Demarches/2484?&view-mode=formulaire-avis&nd_mode=en-ligne-enti%C3%A8rement&nd_source=button&key=b4053638f7a51e868dea83f4361ebc23"
+    img src="https://voxusagers.numerique.gouv.fr/static/bouton-blanc.svg" alt="Je donne mon avis" title="Je donne mon avis sur cette démarche"
```

Comment:
If you add an image from an external website you need to update the `img_src` content policy

Currently I get an error in my console:
> Content Security Policy: Les paramètres de la page ont empêché le chargement d’une ressource à https://voxusagers.numerique.gouv.fr/static/bouton-blanc.svg (« img-src »). Un rapport CSP est en cours d’envoi.